docs/curl-commands.md

```diff
@@ -0,0 +1,33 @@
+# Customs Declarations Curl Commmands
```

Comment:
this should read `Customs Inventory Linking Exports`

Reply:
done